examples/with-urql-graphql-codegen/next.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  webpack: (config) => {
+    config.node = {
+      fs: 'empty'
+    }
+    return config
+  }
+};
```

Comment:
Why is this needed?

Reply:
is the default config when i made the project using the cli